### PR TITLE
Add SECURITY.md

### LINKED ISSUE #346: Set up security policy
The [current security policy page](https://github.com/OpenZeppelin/cairo-contracts/security/policy) is displaying the wrong file. We should set up a proper security policy file including contact information.

### PR DESCRIPTION
**Pull request by nikitastupin**
Fixes #346

#### PR Checklist

- [x] Tests
- [x] Documentation
- [x] Changelog entry

### CODE REVIEW

**Comment by nikitastupin** [comment]
By the way, why `security@openzeppelin.org` and not `.com`?

File: README.md [hunk 1]
```diff
@@ -247,9 +247,7 @@ See [Memoization](docs/Utilities.md#memoization) in the Utilities documentation
 
 ## Security
 
-This project is still in a very early and experimental phase. It has never been audited nor thoroughly reviewed for security vulnerabilities. Do not use in production.
```
Comment (martriay) [inline README.md:250]: Not sure about removing this from the README. Maybe having it in both places?

Comment (nikitastupin) [reply README.md:250]: I was thinking about the same! Let's keep this line in `README.md` 👌

**Review by andrew-fleming — COMMENTED** [review]
Thanks, @nikitastupin! 

> By the way, why `security@openzeppelin.org` and not `.com`?

Can we confirm it's `.org` and not `.com`?

**Comment by ericDeCourcy** [comment]
lets go with `.com`

**Review by andrew-fleming — APPROVED** [review]
Thank you :)

**Pull request merged**